libs/sdk-model/src/insight/tests/sanitization.test.ts

```diff
@@ -81,4 +82,33 @@ describe("insightSanitize", () => {
         expect(resultTotals1).toEqual([grandTotal]);
         expect(resultTotals2).toEqual([grandTotal]);
     });
+
+    it("should remove duplicate drillable bucket attribute items", () => {
+        const targetDrillUri = "/gdc/md/heo9nbbna28ol3jnai0ut79tjer5cqdn/obj/1091";
+        const departmentUri = "/gdc/md/heo9nbbna28ol3jnai0ut79tjer5cqdn/obj/1103";
+        const insight = newInsightDefinition("foo", (m) =>
+            m.buckets([
+                { localIdentifier: "measures", items: [m1, m2] },
+                {
+                    localIdentifier: "attribute",
+                    items: [
+                        modifyAttribute(a1, (a) => a.displayForm(uriRef(targetDrillUri))),
+                        modifyAttribute(a2, (a) => a.displayForm(uriRef(targetDrillUri))),
+                        modifyAttribute(a3, (a) => a.displayForm(uriRef(departmentUri))),
+                    ],
+                },
+            ]),
+        );
+        const sanitized = insightSanitize(insight);
+        const result = insightBucket(sanitized, "attribute");
+        const expectedResult: IBucket = {
+            localIdentifier: "attribute",
+            items: [
+                newAttribute(uriRef(targetDrillUri), (a) => a.localId("a_label.account.id")),
+                newAttribute(uriRef(departmentUri), (a) => a.localId(Department.attribute.localIdentifier)),
+            ],
+        };
+
+        expect(result).toEqual(expectedResult);
```

Comment:
you can use expect(result).toMatchSnapshot() a d let the computer do typing of the expectedResult for you :)

Reply:
added snapshot